### ISSUE TITLE
added potentially missing `aria-`-HTML-attributes

### ISSUE THREAD

**Issue by mfranzke** [db-ux-design-system/db-ux-design-system.github.io#66]
> in case that this `button` actually has an influence on elements being displayed on the page, we would actually need [`aria-expanded`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-expanded)- and [`aria-controls`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-controls)-HTML-attributes. 

 _Originally posted by @mfranzke in [283dbca](https://github.com/db-ux-design-system/db-ux-design-system.github.io/commit/283dbca23782a892fa4ba5a23b67f8ff813e3472#r176748920)_
